README.adoc

```diff
@@ -59,7 +59,7 @@ endif::[]
 :image-uri-screenshot: https://raw.githubusercontent.com/asciidoctor/asciidoctor/master/screenshot.png
 
 {uri-project}[Asciidoctor] is a _fast_ text processor and publishing toolchain for converting {uri-what-is-asciidoc}[AsciiDoc] content to HTML5, DocBook, PDF, and other formats.
-Asciidoctor is written in Ruby, packaged and distributed as a gem to {uri-rubygem}[RubyGems.org], and packaged for popular Linux distributions, including Fedora, Debian, Ubuntu, and Alpine.
+Asciidoctor is written in Ruby, packaged and distributed as a gem to {uri-rubygem}[RubyGems.org], and packaged for popular Linux distributions, macOS, and Windows.
```

Comment:
Technically, it's not true that Asciidoctor is packaged for Windows. There is a chocolatey package for AsciidoctorJ, but not Asciidoctor core. However, it can be installed on Windows using `gem install`.